worker-document-interface/src/main/java/com/hpe/caf/worker/document/model/Scripts.java

```diff
@@ -42,6 +42,13 @@
     @Nonnull
     Script add(int index);
 
+    /**
+     * Closes the bindings and associated context all of the scripts in this list.
+     * <p>
+     * This is equivalent to calling {@link Script#closeBindings() closeBindings()} for each of the scripts in the list.
+     */
+    void closeBindings();
```

Comment:
Surely closing the bindings is entirely internal.  Why would we be exposing something like this on the external interface used by workers?